### PR TITLE
Simplify Docusaurus template

### PR DESCRIPTION
**Pull request by maoo**
@mcleo-d @grizzwolf - while working on datahub website, I realised that a lot of the Docusaurus template logic is not needed when creating a new website, and it takes time to cleanup and remove unnecessary stuff.

I've removed these items (most of which was based on FDC3, which we can always check in the future for reference) and cleaned things up a bit.

With this structure, not only my work will be easier during the contribution process, but I'm also more confident that contributors will be able to create the initial website more easily, without the need of our assistance.

WDYT? Happy to merge?

### CODE REVIEW

File: website/core/Footer.js [hunk 1]
```diff
@@ -109,6 +93,13 @@ class Footer extends React.Component {
                 </a>
               </div>
             )}
+            <div className="social">
+              <a
+                href={`https://www.linkedin.com/company/finosfoundation`}
+                className="linkedin-follow-button">
+                FINOS on LinkedIN
```
Comment (mcleo-d) [inline website/core/Footer.js:100]: @maoo - `LinkedIN` should be `LinkedIn` 

👍

File: .gitignore [hunk 2]
```diff
@@ -8,5 +8,6 @@ lib/core/MetadataBlog.js
 website/translated_docs
 website/build/
 website/yarn.lock
+website/package-lock.json
```
[the remark this answers is not in the document]
Comment (maoo) [reply .gitignore:11]: Good point, thought about it... but remember, this is a repo template, and therefore there should not be any type of generated files.

This makes me think that also contributing.md should be out (and in .gitignore).

File: website/core/Footer.js (hunk 1 above)
Comment (maoo) [reply website/core/Footer.js]: Fixed!

File: .gitignore (hunk 2 above)
[the remark this answers is not in the document]
Comment (mcleo-d) [reply .gitignore:11]: Good point! - Does this mean the projects that are consuming Docusaurus into their repos will need to add `website/package-lock.json` into the `.gitignore` of their project root?

[the remark this answers is not in the document]
Comment (maoo) [reply .gitignore:15]: No, not really, but it's a good point, I've updated the `.gitignore` file, see https://github.com/finos/project-blueprint/blob/simplify-website/.gitignore

Please note that we use YARN to build, so yarn.lock should be in the source code, but not package-lock.json .

Does it make sense?

[the remark this answers is not in the document]
Comment (mcleo-d) [reply .gitignore:15]: That makes perfect sense! 😀 👍

**Pull request merged**